src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec

```diff
@@ -24,6 +24,12 @@ OData .NET library is open source at http://github.com/OData/odata.net</descript
     </frameworkAssemblies>
   </metadata>
   <files>
+    <file src="$ProductRoot$\Desktop\Microsoft.OData.Client.dll" target="lib\net45" />
+    <file src="$ProductRoot$\Desktop\Microsoft.OData.Client.xml" target="lib\net45" />
+    <file src="$ProductRoot$\Desktop\Microsoft.OData.Client.pdb" target="lib\net45" />
+    <file src="$ProductRoot$\.NETPortable\v4.5\Microsoft.OData.Client.dll" target="lib\portable-net45+win8+wpa81" />

```

Comment:
Let's just do the net45, Netstandard1.1 and portable-net45+win8+wpa81 should be equivalent.

Reply:
Woops, I jumped the gun there.